litestar/contrib/mongodb_motor/repository/types.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any, MutableMapping, TypeVar
+
+from motor.motor_asyncio import AsyncIOMotorCollection
+
+__all__ = ("Document", "AsyncMotorCollection")
+
+
+Document = TypeVar("Document", bound=MutableMapping[str, Any])
+AsyncMotorCollection = AsyncIOMotorCollection
```

Comment:
This is redundant - you can simply import this type from the source where you need it, rather than obfuscate stuff by re-exporting